subt_ros/launch/competition_init.launch

```diff
@@ -1,6 +1,8 @@
 <?xml version="1.0"?>
 <launch>
 
+  <env name="IGN_TRANSPORT_TOPIC_STATISTICS" value="1"/>
```

Comment:
Probably not needed here as well.

Reply:
I think this is needed in order get topic statistics on bridge, I'll double check.